pkg/kubelet/container_bridge.go

```diff
@@ -82,6 +82,16 @@ func ensureCbr0(wantCIDR *net.IPNet) error {
 		}
 		return createCBR0(wantCIDR)
 	}
+	// Put the container bridge into promiscuous mode to force it to accept hairpin packets.
```

Comment:
We will unless there's an error: https://github.com/kubernetes/kubernetes/pull/21326/files#r53406092, or did you mean something else?